packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -8,10 +8,11 @@ type TItemWrapper = {
 };
 type TRadioGroup = {
     className?: string;
-    name: string;
+    name?: string;
```

Comment:
Is there a specific reason we're making `name` optional? I suppose `name` should always be required in a controlled form.

Reply:
Changed this